Gemfile

```diff
@@ -1,9 +1,10 @@
 source 'https://rubygems.org'
 
 gemspec
-gem 'test-kitchen'
-gem 'kitchen-sync'
+gem 'hashie'
```

Comment:
should this not be added to the gemspec so it gets installed as a dependency of kitchen-salt when it is `gem install`ed?

Reply:
I'm confused. I just realized there are dependencies both in the `Gemfile` and `.gemspec`. Would you like me to refactor that at the same time and ditch the `Gemfile`?

Reply:
Yeah, i am just getting back into ruby recently since we took over this project, so would appreciate any help with getting that all organized that I can git :D